✨ Added Int.down(to:) helper function (#670)
SwifterSwift/SwifterSwift: CHANGELOG.md

## Upcoming Release

### Added
- **Int**

>   - `down(to:)` extension for Int which iterates from this number down to the given number, inclusive, decrementing by one each time.

Reviewer: Add the PR number and your name, like in the other entries.

> - `down(to:)` extension for Int which iterates from this number down to the given number, inclusive, decrementing by one each time. . [#670](https://github.com/SwifterSwift/SwifterSwift/pull/670) by [cHaLkdusT](https://github.com/cHaLkdusT).

Reviewer: ```suggestion
- `down(to:)`, which iterates from the receiver down to the given value, inclusive, decrementing by one each time. [#670](https://github.com/SwifterSwift/SwifterSwift/pull/670) by [cHaLkdusT](https://github.com/cHaLkdusT).
```
